app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ForgotPasswordsController < ApplicationController
+
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
You should use `find_by` instead of `where(args).first` - as `find_by` is more effective and preferred for single record search.